health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/UploadDataCustom.js

```
@@ -0,0 +1,1125 @@
+          if (property?.isRequired && required.indexOf(property?.name) === -1) {
+            required.push(property?.name);
+          }
+
+          // If orderNumber is missing, default to a very high number
+          columns.push({ name: property?.name, orderNumber: property?.orderNumber || 9999999999 });
+        }
+      }
+    }
+    enrichSchema(convertData, properties, required, columns);
+    const newData = JSON.parse(JSON.stringify(convertData));
+    delete newData.campaignType;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)